internal/storage/postgres.go

```diff
@@ -0,0 +1,310 @@
+package storage
+
+import (
+	"context"
+	"github.com/jackc/pgx/v4"
+	"github.com/pkg/errors"
+	"github.com/zamibaru89/gophermart/internal/config"
+	"log"
+)
+
+type PostgresStorage struct {
+	Connection *pgx.Conn
+}
+
+func NewPostgresStorage(c config.ServerConfig) (Repo, *pgx.Conn, error) {
+	conn, err := pgx.Connect(context.Background(), c.DSN)
```

Comment:
а еще можно коннект создать выше и передать сюда, тогда не придется его возвращать)
```suggestion
func NewPostgresStorage(conn *pgx.Conn) (Repo, error) {
```